server/api.go

```diff
@@ -47,6 +53,66 @@ type SessionResult struct {
 	LegacySession bool `json:"-"` // true if request was started with legacy (i.e. pre-condiscon) session request
 }
 
+// SessionDisclosureResult contains an extended array of generalized credentials with the zkp
+type SessionDisclosureResult struct {
+	Identifier  *big.Int                            `json:"identifier"` // Identifier assigned by verifier
```

Comment:
What is the purpose of this? If the requestor can choose the nonce itself, could it not use the nonce instead of this to identify the session?

Reply:
The identifier can differ from the nonce in our infrastructure. We have internal identifiers that identify the request, but a third party can assign a nonce through our APIs.